imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -861,10 +848,19 @@
    },
    "outputs": [],
    "source": [
-    "%%bash -s {project_id} {location} {dataset_id} {inference_dicom_store_id} {tcia_api_key} {input_mammo_study_uid}\n",
+    "from google.cloud import storage\n",
     "\n",
-    "# Store input mammo image into Cloud Healthcare DICOMWeb API.\n",
-    "python -m scripts.store_tcia_in_hc_api --project_id=$1 --location=$2 --dataset_id=$3 --dicom_store_id=$4 --tcia_api_key=$5 --has_study_uid=$6"
+    "http = httplib2.Http()\n",
```

Comment:
Do you think you can move this to another change? This needs a deeper discussion on how to fit this in

Reply:
already reverted https://github.com/GoogleCloudPlatform/healthcare/issues/433